play-json/jvm/src/main/scala/play/api/libs/json/jackson/StringBasedNumericNode.scala

```diff
@@ -0,0 +1,46 @@
+/*
+ * Copyright (C) from 2022 The Play Framework Contributors <https://github.com/playframework>, 2011-2021 Lightbend Inc. <https://www.lightbend.com>
+ */
+
+package play.api.libs.json.jackson
+
+import com.fasterxml.jackson.core.{ JsonGenerator, JsonParser, JsonToken }
+import com.fasterxml.jackson.databind.SerializerProvider
+import com.fasterxml.jackson.databind.node.NumericNode
+
+import java.math.{ BigDecimal, BigInteger }
+
+/**
+ * A numeric node that is represented as a string.
+ *
+ * For internal use only. Some methods are not implemented and will throw an exception.
+ */
+private[jackson] case class StringBasedNumericNode(text: String) extends NumericNode {
+
+  override def numberType: JsonParser.NumberType = JsonParser.NumberType.BIG_DECIMAL
+
+  override def asText(): String = text
+
+  override def asToken(): JsonToken = JsonToken.VALUE_NUMBER_FLOAT
+
+  override def isIntegralNumber: Boolean = false
+
+  override def serialize(jgen: JsonGenerator, ctxt: SerializerProvider): Unit =
+    jgen.writeNumber(text)
+
+  override def numberValue: Number = throw new NotImplementedError
```

Comment:
So doesn't look like a `NumericNode` for now

Reply:
I describe in the PR description as to why I don't need this method to be implemented. I only really need the serialize method to be implemented. This is not a general purpose public class - it is an internal class written this way to avoid extra conversions.